velox/connectors/hive/HiveConfig.h

```diff
@@ -103,9 +103,10 @@ class HiveConfig {
   /// The max coalesce bytes for a request.
   static constexpr const char* kMaxCoalescedBytes = "max-coalesced-bytes";
 
-  /// The max coalesce distance bytes for combining requests.
-  static constexpr const char* kMaxCoalescedDistanceBytes =
-      "max-coalesced-distance-bytes";
+  /// The max merge distance combining read requests.
+  static constexpr const char* kOrcMaxMergeDistance = "orc-max-merge-distance";
```

Comment:
This config should be format agnostic.  Do we have this name just to be compatible with Presto?  If so we can keep it like this: `kMaxCoalescedDistanceBytes = "orc-max-merge-distance"`

Reply:
@Yuhta 
It is a good question. I spent some time trying to think what we should do best.

For connectors we fetch config properties from the config file.
Then, often, such values can be overridden by a session property.

I checked again and Presto has:
`"hive.orc.max-merge-distance"` as config property (specified in the config file).
`"orc_max_merge_distance"` as session property (can be specified per session or query).
We want to ensure that the session property takes effect.
But it would also be good if Presto C++ could pick the config property properly as well.

Hence, renaming the old `kOrcMaxMergeDistance` config property. But I didn't do it right. Instead of `"max-coalesced-distance-bytes"` we should use `"hive.orc.max-merge-distance"`.

Reply:
Can we keep it as `max-coalesced-distance-bytes` in Velox and do the translation in Prestissimo?